src/main/kotlin/com/falcon/falcon/core/security/JwtFilter.kt

```diff
@@ -40,3 +66,8 @@ private fun String.extractToken(): String? {
     // e.g Bearer 1234token
     return this.split(" ")[1]
 }
+
+private fun String.isUrlPath(): String? = this.takeIf { this == "/url" }
```

Comment:
Pq q vc retorna null ao inves de retornar um boolean (return this == "/url") ou até um (return this.startsWith("/url")
Inclusive de onde vem isso?